src/gridGL/interaction/keyboard/keyboardClipboard.ts

```diff
@@ -28,7 +28,7 @@ export function keyboardClipboard(props: {
   }
 
   // Command + Shift + C
-  if ((event.metaKey || event.ctrlKey) && event.shiftKey && event.code === 'KeyC') {
+  if ((event.metaKey || event.ctrlKey) && event.key === 'C') {
```

Comment:
```suggestion
if ((event.metaKey || event.ctrlKey) && event.shiftKey && event.code.toLowerCase() === 'c') {
```

For a bit more readable code.